### DISCUSSION TITLE
Vite project in the turbo repo setup is not building

### DISCUSSION THREAD

**Discussion by mayank1513** [vercel/turborepo#8011] [Help] [answered]
### Summary

Please refer to this branch - https://github.com/react18-tools/turborepo-template/tree/loaders

The nextjs-example and remix-example builds well. However, the vite-example throws error. 
```
RollupError: ../../packages/shared-ui/src/root-layout.tsx (12:9): "SimpleLoader" is not exported by "../../lib/fork-me/dist/server/index.js", imported by "../../packages/shared-ui/src/root-layout.tsx".
```

This error makes no sense as SimpleLoader component is exported by the `dist/server/index.js` file and the other 2 examples build well using that file.

### Example

https://github.com/react18-tools/turborepo-template/tree/loaders

**Comment by anthonyshew** [comment, marked as answer]
This doesn't appear to be a Turborepo error. You may want to check the Vite or Rollup communities for folks running into similar issues. If I were to guess, the module is not being resolved correctly due to some missing configuration.

**Comment by mayank1513** [comment]
Fixed it already. Sorry, forgot to close this issue.
